MIS_Recode.do

```diff
@@ -157,10 +157,10 @@ save `hm',replace
 ************************************
 *****domains using hh level data****
 ************************************
-use "${SOURCE}/DHS-`name'/DHS-`name'hm.dta", clear
+use "${SOURCE}/MIS-`name'/MIS-`name'hm.dta", clear
     rename (hv001 hv002 hvidx) (v001 v002 v003)
 
-    merge 1:m v001 v002 v003 using `birth'
+    merge 1:m v001 v002 v003 using "${SOURCE}/MIS-`name'/MIS-`name'birth.dta"
```

Comment:
Could you please clarify why changing the merge data source? Here the command is trying to merge the temp data generated in command line 106. You are changing this to the raw data instead of using the data generated in the previous section of the code. @imarabbit

Reply:
I changed the merge data source because when I tried merging the temp data, an error "variable v001 not found" is reported. I have checked the do file and the data but I was not able to locate the problem and solve it (variable v001 is supposed to exist after variable hv001 in the birth temp data being renamed as v001). So I had to change the source to ensure other indicators besides those relating to household level can be generated successfully,  and then reported this problem in the pull request description. By the way, this problem also appears when I deal with Angola 2006-07 dataset. Could you please show me how to fix it? Thanks a lot in advance!

Reply:
Thanks @imarabbit please @ me here once the modification is completed.